### PR TITLE
Measurements Map

### PR DESCRIPTION
**Pull request by ZZwatermelon**
Implements a map addressing issue #3

### CODE REVIEW

File: proc/vcm.go [hunk 1]
```diff
@@ -51,38 +56,30 @@ func ParseConfig(filename string) (*Configuration, error) {
 	}
 
 	// Set default values for measurements if not specified
-	for i := range GswConfig.Measurements {
+	for k := range GswConfig.Measurements {
```
Comment (AarC10) [inline proc/vcm.go:59]: Why did you switch the var?

**Review by AarC10 — APPROVED** [review]
Great! Looks like simulating sensor module and receiving its telemetry still works
image

File: proc/vcm.go (hunk 1 above)
Comment (ZZwatermelon) [reply proc/vcm.go:59]: originally, I had it as
`for k, v := range`
as I was using the v for debugging
and generally imo the k makes more sense, as it is iterating through the keys of the map, whereas before the i made sense as it was iterating through the indexes of the list

Comment (AarC10) [reply proc/vcm.go:59]: Fair

**Pull request merged**